controllers/drcluster_controller_test.go

```diff
@@ -186,6 +186,42 @@ func inspectClusterManifestSubscriptionCSV(match bool, value string, drcluster *
 	}
 }
 
+func updateDRClusterParameters(clusterName string, clusterFence ramen.ClusterFenceState,
+	cidrs []string, s3ProfileName string,
+) *ramen.DRCluster {
+	drclusterLookupKey := types.NamespacedName{
+		Name: clusterName,
+	}
+	latestdrc := &ramen.DRCluster{}
+
+	retryErr := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
+		var err error
+
+		err = apiReader.Get(context.TODO(), drclusterLookupKey, latestdrc)
+		if err != nil {
+			return err
+		}
+
+		latestdrc.Spec.ClusterFence = clusterFence
+
+		if s3ProfileName != "" {
+			latestdrc.Spec.S3ProfileName = s3ProfileName
+		}
+
+		if cidrs != nil {
+			latestdrc.Spec.CIDRs = cidrs
+		}
+
+		err = k8sClient.Update(context.TODO(), latestdrc)
```

Comment:
This may not be needed if the new fields are equal to the latest version. It would be better to update only if something has changed.

Reply:
This is the point of the test - we always changing something in order to test it. So, nothing is changed - not our siituation

Reply:
We always update all fields? This code cannot be reused by future tests that will
update only some of the fields?

Reply:
We always update at least one of these fields. Otherwise, this function is not invoked. New test cast - changing something - updating drcluster.